src/collection/views/CollectionDetail.tsx

```diff
@@ -51,7 +51,8 @@ import { WORKSPACE_PATH } from '../../workspace/workspace.const';
 import { COLLECTION_PATH } from '../collection.const';
 import { DELETE_COLLECTION, GET_COLLECTION_BY_ID } from '../collection.gql';
 import { ContentTypeString, DutchContentType, toEnglishContentType } from '../collection.types';
-import { FragmentDetail, ShareCollectionModal } from '../components';
+import { ShareCollectionModal } from '../components';
+import FragmentListDetail from '../components/fragment/FragmentListDetail';
```

Comment:
Kan je FragmentListDetail niet in de index file van `../components` plaatsen? Dan kan je ze samen via die erboven importen.